packages/react-component-library/src/hooks/useClickMenu.ts

```diff
@@ -22,14 +34,17 @@ export const useClickMenu = (
     if (e.target === attachedToRef.current) {
       e.preventDefault()
       setOpen(true)
+      if (onShow) onShow()
       return
     }
 
     setOpen(false)
+    if (onHide) onHide()
```

Comment:
Maybe worth including an event object arg when invoking these callbacks?

Reply:
Can do if you think it's worth it though the only thing of use is the target and you wouldn't use it